Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/StubIrTextEmitter.kt

```diff
@@ -445,7 +445,7 @@ class StubIrTextEmitter(
     }
 
     private fun renderValueUsage(value: ValueStub): String = when (value) {
-        is StringConstantStub -> value.value
+        is StringConstantStub -> value.value.quoteAsKotlinLiteral()
```

Comment:
How did this work before?

Reply:
Quotes were added in `MacroConstantStubBuilder`. We don't need these quotes in metadata mode, so they were moved to `StubIrTextEmitter`.